lib/engines/content_block_manager/features/edit_object.feature

```diff
@@ -25,6 +25,8 @@ Feature: Edit a content object
     And I should see a back link to the review page
     When I choose to publish the change now
     And I accept and publish
+    Then I am on the confirmation page
```

Comment:
I think I'd prefer `Then I should be taken to the confirmation page` for this step.